charts/mirrorbits-parent/tests/custom_values_ingress_test.yaml

```
@@ -56,27 +59,28 @@ tests:
   - it: should create ingress with multiple hosts and multiple paths with full customized values
     template: ingress.yaml
     set:
-      ingress:
-        className: my-ingress-class
-        annotations:
-          'app.kubernetes/whatever-with-chars': 'same/here-right'
-        hosts:
-          - host: company.org
-            paths:
-              - path: /.*[.](deb|hpi|war|rpm|msi|pkg|sha256|md5sum|zip|gz|pdf|json|svg|sh|jpeg|ico|png|html)$
-                backendService: httpd
-              - path: /
-                backendService: mirrorbits
-                pathType: ImplementationSpecific
-          - host: fallback.company.org
-            paths:
-              - path: /
-                backendService: httpd
-        tls:
-          - secretName: ingress-tls
-            hosts:
-              - company.org
-              - fallback.company.org
+      global:
+        ingress:
+          className: my-ingress-class
+          annotations:
+            'app.kubernetes/whatever-with-chars': 'same/here-right'
+          hosts:
+            - host: company.org
+              paths:
+                - path: /.*[.](deb|hpi|war|rpm|msi|pkg|sha256|md5sum|zip|gz|pdf|json|svg|sh|jpeg|ico|png|html)$
+                  backendService: httpd
+                - path: /
+                  backendService: mirrorbits
```

```suggestion
                  backendService: mirrorbits-lite
```
(Depends on the acceptance or not of the first suggestions above)

---

👎 I believe this value must not depend on the name of the subchart, hence the choice of `mirrorbits` as a name here.